packages/token-lists/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pancakeswap-token-lists",
+  "version": "2.10.0",
```

Comment:
v1 ?

Reply:
@RabbitDoge this is the version of the list we have right now in the exchange